src/cli/translate.ts

```
@@ -344,90 +476,123 @@ export const translateCommand = new Command('translate')
+                const titleTranslation = getTitleTranslation(
+                  sourceFrontmatter.title,
+                  source,
+                  target,
+                )
+                if (titleTranslation) {
+                  newFrontmatter.title = titleTranslation
+                }
+              }
+
+              if (typeof newFrontmatter.title !== 'string') {
+                delete newFrontmatter.title
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)